deploy/helm/sumologic/values.yaml

```diff
@@ -164,6 +164,11 @@ sumologic:
         default:
           name: (default-metrics)
           config-name: endpoint-metrics
+        default-otlp:
+          name: default-metrics-otlp
```

Comment:
Can we standardize the naming? The OTLP source for logs is named `logs-otlp`, can we name this simply `metrics-otlp`?

Reply:
Sure, will do.